packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -158,13 +157,14 @@ async function terraformAirnodeManage(
     ['var', 'configuration_file', configPath ? path.resolve(configPath) : 'NULL'],
     ['var', 'secrets_file', secretsPath ? path.resolve(secretsPath) : 'NULL'],
     ['var', 'handler_dir', handlerDir],
+    ['var', 'disable_concurrency_reservation', `${!!cloudProvider.disableConcurrencyReservations}`],
```

Comment:
Maybe `Boolean(thing).toString()`?

Reply:
That will be even longer. Does it have any advantages in comparison?